Modules/Package.swift

```diff
@@ -134,6 +135,7 @@ enum XcodeSupport {
                 "WordPressFlux",
                 "WordPressShared",
                 "WordPressUI",
+                "WordPressAppUI",
```

Comment:
I'd suggest adding a new auth-related module for these new screens like we had with `WordPressAuthentificator` as opposed to a board "AppUI". For shared components, we already have `WordPressUI`.

Reply:
Removed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23726/commits/ce8d2f860b8e26020693d84ec061a3a8b25ee318.